src/kokkos/test/kokkos/TrackingRecHit2D_t.cc

```diff
@@ -0,0 +1,70 @@
+#include "KokkosCore/kokkosConfigCommon.h"
+#include "KokkosCore/kokkosConfig.h"
+
+#include "CondFormats/pixelCPEforGPU.h"
+#include "KokkosDataFormats/TrackingRecHit2DKokkos.h"
+
+namespace testTrackingRecHit2DKokkos {
+
+  template<typename MemorySpace>
+  void fill(Kokkos::View<TrackingRecHit2DSOAView, MemorySpace> hits) {
+
+    assert(hits.data());
+    auto &hits_ = *hits.data();
```

Comment:
The preferred way in Kokkos would be
```suggestion
    auto &hits_ = hits();
```
(I'm personally fine with either way)